index.bs

```diff
@@ -662,7 +662,7 @@ or {{DOMException}}
 should be used for errors,
 whether they are exceptions, promise rejection values, or properties.
 
-<h2 id="device-apis">Device APIs</h2>
+<h2 id="device-apis">Device and Wrapper APIs</h2>
```

Comment:
It's not clear to me that this warrants merging.  But if it does, it seems like maybe the introduction to the section should be extended to talk about wrapper APIs.  At the very least, I think wrapper APIs deserve a definition given that I don't think it's a well-known term.